docs/section19/19-7.md

```diff
@@ -0,0 +1,84 @@
+                - Check if DCV was modeled: `if terminal.has_demand_control_ventilation == true: was_DCV_modeled_proposed = true`  
+                - Check if the hvac system serving the terminal unit has dcv modeled (weili may have to include some logic to get the hvac object from the id): `if terminal.served_by_heating_ventilating_air_conditioning_system.fan_system.demand_control_ventilation_control == true: was_DCV_modeled_proposed`  
+            
+            - Add to counter: `counter = counter + 1`  
+
+        - Create aggregated OA CFM schedule for the baseline for the zones served by the hvac system: `aggregated_min_OA_schedule_across_zones_b = aggregate_min_OA_schedule_across_zones(B_RMI,zone_OA_CFM_list_of_schedules_b)`  
+        - Create aggregated OA CFM schedule for the proposed for the zones served by the baseline hvac system: `aggregated_min_OA_schedule_across_zones_p = aggregate_min_OA_schedule_across_zones(P_RMI,zone_OA_CFM_list_of_schedules_p)`  
+
+        - Reset OA_CFM_schedules_match boolean variable to True: `OA_CFM_schedules_match = True`  
+        - Check if schedules mismatch for any hours of the year, if so set boolean to false: `for p to Range(8760):`  
+            - Check if the baseline and proposed schedules mismatch: `if aggregated_min_OA_schedule_across_zones_b[p] != aggregated_min_OA_schedule_across_zones_p[p]: OA_CFM_schedule_match = false`   
+
+        - Sum OA CFM across baseline schedule to get the total for comparison purposes in the rule assertions: `modeled_baseline_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_b)`  
+        - Sum OA CFM across proposed schedule to get the total for comparison purposes in the rule assertions: `modeled_proposed_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_p)`  
+
+
+
+        - **Rule Assertion:** 
+        - Case 1: If the modeled baseline and proposed minimum OA cfm values are identical then pass: `if OA_CFM_schedule_match == true: outcome = "PASS"`  
+        - Case 2: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has demand controlled ventilation modeled in the proposed (Exception 1) then the outcome is Fail: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and was_DCV_modeled_baseline == false and was_DCV_modeled_proposed == true: outome = "FAIL" and raise_message "For <insert hvac.id> the baseline modeled minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears that demand-controlled ventilation was modeled in the proposed and not the baseline model and that demand-controlled ventilation may be double accounted for in the model (per the HVAC controls and via reduced OA CFM rates in the proposed)."`  
```

Comment:
I think you should add condition "and zone_air_distribution_effectiveness_greater_than_1 == false" before ":outcome="FAIL""

Reply:
Done